build/litsearch/combineDataSets.py

```diff
@@ -0,0 +1,96 @@
+import pandas as pd
+import os
+import urllib.request
+import requests
+import datetime as dt
+
+def update_AI(fname):
+    # Download most up-to-date metadata from Allen AI COVID-19 dataset
+    url = 'https://ai2-semanticscholar-cord-19.s3-us-west-2.amazonaws.com/latest/metadata.csv'
+    urllib.request.urlretrieve(url, filename=fname)
+
+def clean_df(df):
+    df = df.dropna(subset=["doi"])
+    return df
+
+def paper_info(row):
+    """Queries metadata from bioRxiv or medRxiv based on paper DOI
+    Accepts: Row formatted as review_sources df, must have "doi" and "publication"
+    Returns: Row, updated as possible from *arxiv API """
+    if row["publication"] in ["bioRxiv", "medRxiv"]:
+        # Format URL for *rxiv API and convert metadata to dict from json response
+        pub = row["publication"].lower()
+        doi = row["doi"]
+        url = "https://api.biorxiv.org/details/{0}/{1}".format(pub,doi)
+        r = requests.get(url)
+        paper_data = r.json()
+        metadata = dict(paper_data["collection"][0])
+
+        # Update the row with metadata retrieved
+        row["URL"] = url
+        for kwd in ["title", "authors", "date", "abstract"]:
+            row[kwd] = metadata[kwd]
+    return row
+
+if __name__ == "__main__":
+    # Update external data, if not updated yet today, and read
+    metadata_fname = "./output/AllenAI-metadata.csv"
+    if not os.path.exists(metadata_fname) or \
+            dt.datetime.fromtimestamp(os.path.getctime(metadata_fname)).date() != \
+            dt.datetime.now().date():
+        print("Update Allen AI metadata")
+        update_AI(metadata_fname)
+    AIA_metadata = pd.read_csv(metadata_fname, low_memory=False)
+    AIA_metadata = clean_df(AIA_metadata)
+    print("AIA has {0} sources".format(len(AIA_metadata)))
+
+    # Update internal dataset, which is maintained with CI
+    review_sources = pd.read_csv("./output/sources_cross_reference.tsv",
+                                 sep="\t")
+    review_sources = clean_df(review_sources)
+    review_sources.rename({"URL":"SecondaryURL"}, inplace=True, axis=1)
+    print("COVID-19 review has {0} sources".format(len(review_sources)))
```

Comment:
```suggestion
    print("COVID-19 review has examined {0} sources".format(len(review_sources)))
```
Very true! Hopefully this disambiguates a bit? Also open to other suggestions about how to describe it!